src/config/containers-and-vms/lxc.md

```diff
@@ -114,6 +119,40 @@ access to other containers. To isolate containers from each other, alter the
 create each container. Trying to fix permissions on a container created with the
 wrong map is possible, but inconvenient.
 
+## Incus
+
+Incus provides an alternative interface to LXC's `lxc-*` utilities. However, it
+does not require the configuration described in [the previous section](#lxc).
+
+Edit `/etc/rc.conf` and set the `CGROUP_MODE` variable to `unified`. Install the
+`incus` package, and [enable](../services/index.md#enabling-services) the
+`incus` service.
+
+Incus users must be belong to the `_incus-admin` group.
```

Comment:
what are "incus users"? users that can perform admin actions in incus?

Reply:
Upstream docs do say that the `_incus-admin` group should only be given to users who should be able to do system-wide incus operations without root. The normal recommended way to use incus as a normal user (other than setup of the daemon) is with the plain `_incus` group.

Reply:
@classabbyamp yes, that's what I meant by incus users. all actions are admin action in normal operation as it's roughly a hypervisor. Open to rephrasing.

@0x5c I don't think that's a fair characterization The normal way is to have access to everything in the incus admin. adding to just the incus group gets a user access to just a personalized project space as described here, as an optional isolation strategy. 

https://linuxcontainers.org/incus/docs/main/explanation/projects/#projects-confined

Having an admin is essential, having limited access is optional, is my view I wrote into the docs. 


or riffed on briefly here 

https://linuxcontainers.org/incus/docs/main/tutorial/first_steps/

    "incus allows basic user access, no configuration and all actions restricted to a per-user project."

   " incus-admin allows full control over Incus."

Reply:
The [upstream documentation page for install instructions](https://linuxcontainers.org/incus/docs/main/installing/) has this, which points out that only root access is actually needed to perform system-wide operations
> The root user and all members of the incus-admin group can interact with the local daemon.

It also includes
> :fire: Important
> Local access to Incus through the Unix socket always grants full access to Incus. This includes the ability to attach file system paths or devices to any instance as well as tweak the security features on any instance.
> Therefore, you should only give such access to users who you’d trust with root access to your system.

Additionally, the [About Security](https://linuxcontainers.org/incus/docs/main/explanation/security/) page states
> - Restrict access to the Incus daemon and the remote API.
> - Do not use privileged containers unless required. If you use privileged containers, put appropriate security measures in place. See the [LXC security page](https://linuxcontainers.org/lxc/security/) for more information.

The above mentioned LXC Security page has this to say about privileged and unprivileged containers
> The former can be thought as old-style containers, they're not safe at all and should only be used in environments where unprivileged containers aren't available and where you would trust your container's user with root access to the host.

Critically, `_incus-admin` is equivalent to unrestricted passwordless sudo access.

While I agree that the `_incus-admin` group should be mentioned, saying that users "must" be in that group does not sound like a reasonable *default* recommendation, especially when prefixing admin commands with `sudo`/`doas` would do the same without the problem of granting passwordless root access.
In other words, while the Void documentation should not be there to criticise security choices a user could deem safe for their setup, the documentation shouldn't be making the assumption of what's "super safe enough" in what it recommends, especially when upstream is quite clear about the risks.

To me the proper way to handle this is to indicate that root access in needed to perform incus administrative operations, with a mention of `_incus-admin` as an alternative along with its caveats. Further details may possibly be more appropriate for the upstream documentation, which can be linked to.

Reply:
that makes sense to me

Reply:
Sound good to me